parsers/src/main/scala/uk/gov/ons/addressIndex/parsers/AddressParser.scala

```diff
@@ -12,6 +12,10 @@ import uk.gov.ons.addressIndex.crfscala.{CrfAggregateFeature, CrfFeature, CrfFea
   */
 object AddressParser extends CrfParser {
 
+  val currentDirectory = new java.io.File(".").getCanonicalPath
+  val modelPath = s"$currentDirectory/parsers/src/main/resources/addressCRFA.crfsuite"
```

Comment:
This value is moved to config eg: https://github.com/ONSdigital/address-index-api/pull/77/files